### ISSUE TITLE
symfony/string v6 required

### ISSUE THREAD

**Issue by bobvandevijver** [dustin10/VichUploaderBundle#1329]
|    Q            |   A
|---------------- | ------
| Bundle version  | 2.0.0 
| Symfony version | 5.4
| PHP version     | 8.1

### Support Question

The 2.0 release drops all support for Symfony < 5.4, but due to the `symfony/string` version that is being required it isn't installable when using a common flex constraint for Symfony: 

```
"extra": {
        "symfony": {
            "require": "5.4.*"
        }
}
```

Is this intentional?

**Comment by garak** [comment]
No, it's a mistake.
Could you open a PR to fix it?

**Comment by garak** [comment]
Released 2.0.1 with your fix. Thanks!
